pom.xml

```diff
@@ -172,7 +172,7 @@
     <fasterxml.jackson.version>2.12.2</fasterxml.jackson.version>
     <snappy.version>1.1.8.4</snappy.version>
     <netlib.java.version>1.1.2</netlib.java.version>
-    <netlib.ludovic.dev.version>1.3.2</netlib.ludovic.dev.version>
+    <netlib.ludovic.dev.version>2-SNAPSHOT</netlib.ludovic.dev.version>
```

Comment:
Ideally we don't depend on a snapshot version - fine if this is in progress and 2.0 will be the final version referenced here.

Reply:
As soon as this PR is good to merge, I’ll release 2.0.0 of dev.ludovic.netlib, change it here and remove the references to the Snapshot repositories.